textproc/recode/Portfile

```diff
@@ -24,16 +24,26 @@ long_description    This recode program has the purpose of converting files \
 
 github.tarball_from releases
 
-depends_test        port:py27-cython
+set python.version  37
+set python.branch   [string range ${python.version} 0 end-1].[string index ${python.version} end]
```

Comment:
It would be simpler to set `python.branch 3.7` and compute `python.version` from it by removing the periods. Also, unless there's a reason not to, let's use Python 3.8 since that's the latest stable version.

Reply:
I used python 3.7 because many ports use that as default and in this way we avoid installing a new version.
From my point of view there should be a general agreement on which version of python should be used and moved to the next atomically (if not technically reason exists on the port).